packages/manager/modules/exchange/src/account/update/account-update.html

```diff
@@ -1,512 +1,880 @@
+                        ></small>
                     </div>
-                    <small
-                        class="help-block"
-                        data-translate="exchange_tab_ALIAS_taken_error_message"
-                        data-ng-if="ctrl.takenEmailError"
-                    ></small>
-                </div>
 
-                <div
-                    class="oui-field mb-4"
-                    data-ng-if="ctrl.newAccountOptions.availableTypes.length > 1"
-                >
-                    <label
-                        class="oui-field__label oui-label required"
-                        for="selectedAccountType"
-                        data-translate="exchange_ACTION_update_account_step1_type_label"
-                    ></label>
-                    <div class="oui-field__content">
-                        <div class="oui-field-control oui-field-control_3">
-                            <div class="oui-select">
-                                <select
-                                    class="oui-select__input"
-                                    id="selectedAccountType"
-                                    name="selectedAccountType"
-                                    required
-                                    data-ng-model="ctrl.selectedAccount.accountLicense"
-                                    data-ng-options="ctrl.services.exchangeAccountTypes.getDisplayValue(type) for type in ctrl.newAccountOptions.availableTypes track by type"
-                                    data-ng-disabled="ctrl.selectedAccount.outlook"
-                                ></select>
+                    <!--Email address-->
+                    <div
+                        class="form-group"
+                        data-ng-class="{'has-error': $ctrl.updateAccountForm.selectedAccountAddress.$dirty && ($ctrl.updateAccountForm.selectedAccountAddress.$invalid || $ctrl.takenEmailError)}"
+                    >
+                        <label
+                            class="control-label required"
+                            for="selectedAccountAddress"
+                            data-translate="exchange_ACTION_update_account_step1_email_label"
+                        ></label>
+                        <div class="input-group">
+                            <input
+                                type="text"
+                                class="form-control"
+                                id="selectedAccountAddress"
+                                maxlength="256"
+                                name="selectedAccountAddress"
+                                required
+                                data-ng-change="$ctrl.checkTakenEmails()"
+                                data-ng-disabled="!$ctrl.canChangePrimary()"
+                                data-ng-model="$ctrl.selectedAccount.login"
+                                data-ng-pattern="/^(?:[a-z0-9]+(?:[-_][a-z0-9]+)*)(?:(?:\.|\+)(?:[a-z0-9]+(?:[-_][a-z0-9]+)*))*$/"
```

Comment:
Extracting this from HTML would ease maintenance

Reply:
Some things are comming from previous implementation (not implemented).